grammar.js

```diff
@@ -120,13 +120,14 @@ module.exports = grammar({
 
 		_label_reference: ($) => seq('&', field('label', $.label_identifier)),
 
-		_node_reference: ($) =>
+		path_node: ($) =>
 			seq(
-				'&{',
-				field('path', $.node_identifier),
-				field('address', optional(seq('@', $.unit_address))),
-				'}'
+				'/',
+				field('name', alias($._node_or_property, $.identifier)),
+				optional(seq('@', field('address', $.unit_address)))
 			),
+		path_in_tree: ($) => repeat1($.path_node),
+		_node_reference: ($) => seq('&{', alias($.path_in_tree, $.path), '}'),
```

Comment:
I'm not sure I understand this part. Is there a reason that "path_in_tree" is being aliased to "path" instead of just being named "path" to begin with?

Reply:
My initial idea was to use a separate rule for paths to nodes to avoid potential overlap with paths in includes, but it seems like string_literal works fine there, so I removed this alias.